### PR TITLE
Add a Vagrantfile and Ansible role for FMN development

### PR DESCRIPTION
**Pull request by jeremycline**
This Vagrantfile and Ansible role sets up a development environment in a
virtual machine to make hacking on FMN easy. It adds a set of systemd
service files that can launch all the necessary development services
under for the user under  systemd which greatly eases certain tasks
like restarting all the services, collecting and filtering logs, etc.

Signed-off-by: Jeremy Cline jeremy@jcline.org

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,6 +38,30 @@ components can be found here:
...
+A Vagrantfile is provided as ``Vagrantfile.example``. You can get started by
+installing Vagrant and starting your own ``Vagrantfile`` from the example::
+
+    $ sudo dnf install ansible vagrant-libvirt vagrant-sshfs
+    $ cp Vagrantfile.example Vagrantfile
+    $ git clone https://github.com/fedora-infra/fmn.rules
+    $ git clone https://github.com/fedora-infra/fmn.lib
+    $ git clone https://github.com/fedora-infra/fmn.consumer
+    $ git clone https://github.com/fedora-infra/fmn.web
+    $ git clone https://github.com/fedora-infra/fmn.sse
```
Comment (pypingou) [inline README.rst]: Out of curiosity, can't we make vagrant do the cloning while at it?

File: ansible/roles/fmn-dev/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    - "fmn.lib"
+    - "fmn.rules"
+    - "fmn.consumer"
+    - "fmn.web"
+    - "fmn.sse"
+  args:
+    chdir: /home/{{ ansible_env.SUDO_USER }}/devel
+
+- name: Enable the RabbitMQ management console
+  command: rabbitmq-plugins enable rabbitmq_management
```
Comment (pypingou) [inline ansible/roles/fmn-dev/tasks/main.yml:96]: didn't know this one, quite cool :)

**Comment by pypingou** [comment]
:+1: for me, looks nice

File: README.rst (hunk 1 above)
Comment (jeremycline) [reply README.rst]: Good idea, I've added that!

**Pull request merged**